docs/sources/operators-guide/tools/mimir-continuous-test.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Grafana mimir-continuous-test"
+menuTitle: "Mimir-continuous-test"
+description: "Use mimir-continuous-test to continuously run smoke tests on live Grafana Mimir clusters."
+weight: 30
+---
+
+# Grafana mimir-continuous-test
+
+Mimir-continuous-test is a standalone tool that you can use to continuously run smoke tests on live Grafana Mimir clusters.
+This tool targets Mimir developers.
+This tool aims to help identifying a class of bugs that could be difficult to spot during development with unit and integration tests.
```

Comment:
```suggestion
Via unit and integration tests, this tool identifies a class of bugs that could be difficult to spot during development.
```

Reply:
Suggestion here is not technically accurate. During development we already use unit and integration tests. This tool helps finding bugs that are difficult to spot during development. I'm going to keep your edit and remove the mention to unit and integration tests.

```suggestion
This tool identifies a class of bugs that could be difficult to spot during development.
```